src/en/getting-started-jaas.md

```diff
@@ -104,29 +90,63 @@ Press the blue button to deploy the changes.
 
 </table>
 
-Adjust your model name, if you desire, then choose a public cloud to deploy to.
+Adjust your model name and choose a public cloud to deploy to.
 Enter your cloud credentials. Scroll down and click Deploy.
 
 ![juju_jaas_choose_cloud](./media/juju_jaas_choose_cloud.png)
 
-## Collaborate
 
-To share your model with another JAAS user you can share the standard Juju way,
-as described in [Users and models][users], or you can share directly from the
-JAAS GUI, like this.
+Your model is now being deployed for you. Now you can fire up the command line
```

Comment:
I'm happy to approve all the change up to this point. I still have problems with the client/install side of things which this solution doesn't answer. perhaps we need to discuss again?